corona-warn-app/cwa-app-android
# Rapid Test Profile UQS navigation (EXPOSUREAPP-9903)

## Fix

Fix back navigation from USQ to Rapid Test Profile

## Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9903)

## How to test
1. Open the app
2. Go to Submission dispatcher
3. Create Rapid Test Profile
4. Press Continue to open UQS
5. Press back button or back arrow in toolbar
6. Verify that you are navigated back to Rapid Test Profile and not to main screen
